product.py

```diff
@@ -558,7 +560,7 @@ def findXenSourceBackups():
     for p in partitions:
         b = None
         try:
-            b = util.TempMount(p, 'backup-', ['ro'], 'ext3')
+            b = util.TempMount(p, 'backup-', ['ro'], diskutil.fs_type_from_device(p))
```

Comment:
Why do we even need to pass file system type here? The kernel should be able to detect the correct type.

Reply:
You're correct that the kernel can figure it out for itself but previously it was explicitly stated so we could pick up any inconsistencies between our expectation of the fs type and the actual fs type
I've kept the structure but since we're asking the partition what the fs type is in this code now it can be removed